### ISSUE TITLE
Not adding an alt-tag does not result in an aria-label with "Loading"

### ISSUE THREAD

**Issue by erikkroes** [PolymerElements/paper-spinner#57]
### Description 

The documents say that there should be an aria-label added when no alt-tag is given. This does not appear to be the case.

The line ''this.alt = this.getAttribute('aria-label') || alt;'' in paper-spinner-behavior.html is reached, but does not result in an aria-label being set.

This means a user of assistive technology has no indication that something is loading.
